featuretools/primitives/standard/transform_primitive.py

```diff
@@ -261,13 +273,53 @@ class Diff(TransformPrimitive):
     uses_full_dataframe = True
     description_template = "the difference from the previous value of {}"
 
+    def __init__(self, periods=0):
+        self.periods = periods
+
     def get_function(self):
         def pd_diff(values):
-            return values.diff()
+            return values.shift(self.periods).diff()
 
         return pd_diff
 
 
+class DiffDatetime(Diff):
```

Comment:
Can you add this to the api_reference under Cumulative Transform Primitives?

Reply:
Done!